packages/frontend/src/router/router.spec.tsx

```diff
@@ -20,7 +20,12 @@ describe('router', () => {
     const routeProvider = app.getSingleton(RouteProvider);
     const parsedUrl = url.parse(path);
 
-    routeProvider.location = { pathname: parsedUrl.pathname, search: parsedUrl.search } as Location;
+    TestRenderer.act(() => {
```

Comment:
I think that this doesn't change anything, right? What was your intention?

Reply:
Newest test renderer requires actions to be in `act` otherwise it causes warnings.